packages/create-keystone-app/lib/copy-example-project.js

```diff
@@ -8,7 +8,7 @@ const createNewProjectFolder = newProjectFolder => {
   fs.mkdirpSync(newProjectFolder);
   const readDir = fs.readdirSync(newProjectFolder);
   if (readDir && readDir.length > 0) {
-    error(`The project directory ${newProjectFolder} is not empty`);
+    error(`The project directory "./${newProjectFolder}" is not empty`);
```

Comment:
There is an edge-case where if passed as an argument, it might not be a relative path. Leading to this error being wrong. Pretty edgy though.

Reply:
Yeah, not too fussed about that I don't think. Will accept a PR in the future if anyone cares enough :-)